stubs/resources/views/flux/sidebar/group.blade.php

```diff
@@ -76,11 +76,13 @@
     <?php endif; ?>
 
 <?php elseif ($heading): ?>
-    <div {{ $attributes->class('block space-y-[2px] in-data-flux-sidebar-collapsed-desktop:hidden') }} data-flux-sidebar-group>
-        <div class="px-3 py-2">
+    <div {{ $attributes->class('block space-y-[2px] in-data-flux-sidebar-collapsed-desktop:pt-2') }} data-flux-sidebar-group>
+        <div class="px-3 py-2 in-data-flux-sidebar-collapsed-desktop:hidden">
             <div class="text-sm text-zinc-400 font-medium leading-none">{{ $heading }}</div>
         </div>
 
+        <flux:separator class="not-in-data-flux-sidebar-collapsed-desktop:hidden" />
+
```

Comment:
Actually there seems to be a vertical layout shift. This change fixes that.

```suggestion
    <div {{ $attributes->class('block space-y-[2px]') }} data-flux-sidebar-group>
        <div class="px-3 py-2">
            <div class="text-sm text-zinc-400 font-medium leading-none in-data-flux-sidebar-collapsed-desktop:hidden">{{ $heading }}</div>

            <div class="not-in-data-flux-sidebar-collapsed-desktop:hidden h-3.5 flex items-center">
                <flux:separator class="" />
            </div>
        </div>

```